lib/pliny/errors.rb

```diff
@@ -108,6 +109,7 @@ class GatewayTimeout < HTTPStatusError; end               # 504
       UnprocessableEntity          => [422, 'Unprocessable entity'],
       TooManyRequests              => [429, 'Too many requests'],
       InternalServerError          => [500, 'Internal server error'],
+      HTTPStatusError              => [500, 'Unspecified'],
```

Comment:
@pedro The original idea here was that `HTTPStatusError` would behave more like an abstract class which would never be instantiated directly; which is also why it didn't get an entry in this table. What do you think of changing this to just raising an `ArgumentError` if the instantiated error class wasn't found in `META`?